data/human/Republic Paradise Threatened.TXT

```diff
@@ -0,0 +1,301 @@
+	on complete
+		event "Analysis Complete"
+
+
+event "Analysis Complete" 5
+		
+
+mission "Return to Ada"
+	minor
+	to offer
+		has "event: Analysis Complete"
+	on offer
+		dialog
+			`A message pings on your console. "The analysis is complete, Captain <last>. Please return to Ada as soon as possible."`
+		log "The analysis of the nukes used in Denebola is complete. Time to find out where they came from."
+	destination "Ada"
+
+
+mission "Investigate the Source A"
+	autosave
+	landing
+	name "Investigate the Source"
+	description "Track down the source of the plutonium used for the Free Worlds nukes."
+	deadline
+	passengers 1
+	priority
+	source "Ada"
+	infiltrating
+	destination "Lodestone"
+	to offer
+		has "Return to Ada: done"
+	blocked `A message comes through from Commander Shah, "Captain <last>, thank you for sticking with us. I have just received word that you will have a passenger for the next part of the mission so please make sure you have room for them."`
+	on offer
+		conversation
+			`As soon as you land a transport pulls up outside of the hatch of your ship and a small contingent of technicians start unloading equipment, supervised by a man in a generic-looking business suit. The man notices you watching and approaches, pulling out a badge identifying him as an agent of Republic Intelligence.`
+			choice
+				`	"Republic Intelligence? Isn't that an oxymoron?"`
+				`	"What's this all about?"`
+					goto friendly
+				`	Wait for him to introduce himself.`
+					goto friendly
+			`	The agent stares blankly at you and as he does so you start to notice just how much muscle is bulking up his jacket. He doesn't blink and eventually you have to look away.`
+			goto friendly
+				label friendly
+			`	"Captain <last>?" the man asks, studying your face intently.`
+			`	"I am Agent Norris, I have been assigned to this case by Parliament. As you can imagine they are very interested in getting results as quickly as possible."`
+			choice
+				`	Nod.`
+				`	"As am I."`
+				`	"I work faster alone."`
+			`	He turns away and waves the technicians past you onto the ship.`
+			`	"They will sweep the vessel for bugs," he says, "it won't take long. As soon as they are done we can be on our way. I will brief you as we go."`
+		log `Agent Norris of Republic Intelligence will tag along as we search for the source of the Wolf Pack nukes.`
+	on enter "Aldebaran"
+		conversation
+			`As soon as you break atmosphere Agent Norris is at your shoulder.`
+			`	"The analysis seems to confirm a Syndicate origin," he says grimly. "There are only a few worlds with plutonium mines where they would have had a chance of avoiding Republic inspectors. Our mission is to get soil samples form each of them."`

```

Comment:
This shows up if you have entered the system, not landed on the planet. So you're not breaking atmosphere.
```suggestion
	on enter "Aldebaran"
		conversation
			`As soon as you enter <system>, Agent Norris is at your shoulder.`
			`	"The analysis seems to confirm a Syndicate origin," he says grimly. "There are only a few worlds with plutonium mines where they would have had a chance of avoiding Republic inspectors. Our mission is to get soil samples form each of them."`
```

Reply:
Actually, I can't say this is right. Don't worry about it for now.